src/lib/protocols/connections/ConnectionService.test.ts

```diff
@@ -185,6 +186,8 @@ describe('ConnectionService', () => {
         })
       );
       expect(connection.invitation).toMatchObject(invitation);
+      expect(connection.alias).toBeUndefined();
+      expect(connectionAlias.alias).toBe('test-alias');
```

Comment:
You already added these tests to the previous PR right?

Reply:
@TimoGlastra No I missed out on the `processInvitation` tests in the previous PR